src/parser.yy

```diff
@@ -173,6 +176,16 @@ void yyerror(bpftrace::Driver &driver, const char *s);
 %type <ast::Program *> program
 
 
+// A pseudo token, which is the lowest precedence among all tokens.
+//
+// This helps us explicitly lower the precedence of a given rule to force shift
+// vs. reduce, and make the grammar explicit (still ambiguous, but explicitly
+// ambiguous). For example, consider the inherently ambiguous `@foo[..]`, which
+// could be interpreted as accessing foo, or indexing into the value of the
```

Comment:
```suggestion
// could be interpreted as an array access of `foo`, or indexing into the value of the
```

Reply:
Hm, I think you have misunderstood this. I tried to clarify, along with the other one.